ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/mixins.py

```diff
@@ -60,6 +60,33 @@ def is_peer_available(self, peer: str) -> bool:
             return False
         return True
 
+    def is_vtep(self, host: str | None = None, *, wan_context: bool = False) -> bool:
```

Comment:
is the `wan_context` used anywhere?

Reply:
No but I think it will be soon and I thought it was weird to just add the Dps check a bit randomly in the function.

Reply:
Fixed